controllers/index.py

```diff
@@ -1,14 +1,20 @@
 import os
 from pathlib import Path
 
-from py_server.config.web import PROJECT_ROOT
+from config.web import PROJECT_ROOT
 
 
 class IndexController:
 
 	def get_view(self):
-		view_file = os.path.join(PROJECT_ROOT, 'py_server/view/index.html')
+		# Use the working directory, not project root
+		# Using project root assumes too many hard coded relative paths from 'dev' root 
+		view_file = os.path.join(os.getcwd(), 'view/index.html')
+		# // Always nervous when reading files directly in controllers  - Offshore it to a service 
```

Comment:
This doesn't make sense at all. I would have to go back to basics to argument on that, better on a call. 

In short, this is not a dynamic value, this is a static path written by the same dev wrote the whole thing. Are you going to "clean" the whole code as well? 😄